tutorial.proto

```diff
@@ -32,37 +58,45 @@ message InlineContent {
 };
 
 message Heading {
-  repeated InlineContent inline_contents = 1;
+  string text = 3;
   int32 level = 2;
+
+  reserved 1;
 };
 
 message Image {
-  // TODO
+  string source = 1;
+  string size = 2;
+  string alt_title = 3;
+  string alt_description = 3;
 };
 
+message YoutubeVideo {
+  Image ImageSource = 1;
```

Comment:
I think using an image for this today is a bit of a hack - if a YouTube video an an image have the same fields required, then maybe we rename the Image message to be "Media" or something like that. If not, then let's create a new message for YouTube videos and use that for videos, and Image for images.